sim/shaman/bloodlust.go

```diff
@@ -60,9 +61,11 @@ func (shaman *Shaman) registerBloodlustCD() {
 		},
 	})
 
-	shaman.AddMajorCooldown(core.MajorCooldown{
-		Spell:    bloodlustSpell,
-		Priority: core.CooldownPriorityBloodlust,
-		Type:     core.CooldownTypeDPS,
-	})
+	if !shaman.IsUsingAPL {
+		shaman.AddMajorCooldown(core.MajorCooldown{
```

Comment:
if using the APL, we want to actually cast the spell, not have it auto used with other MCDs